cobalt/android/cobalt_library_loader.cc

```diff
@@ -0,0 +1,18 @@
+// Copyright 2012 The Chromium Authors
```

Comment:
Needs update.

Reply:
This is a borderline case - normally we don't update this for old files, by default policy. But because we _made a new file_ in principle we could/should.